termd-build-driver/src/test/java/org/jboss/pnc/termdbuilddriver/TermdBuildDriverTest.java

```diff
@@ -151,4 +157,45 @@ public void shouldSetLogsInBuildExecution() throws Exception {
 //        verify(buildExecutionMock, times(1)).clearLogsWebSocketLink();
     }
 
+    @Test(timeout = 60_000)
+    public void shouldFetchFromGitAndBuild() throws Exception {
+        //given
+        Path tmpRepo = Files.createTempDirectory("tmpRepo");
+        String repoPath = "file://" + tmpRepo.toAbsolutePath().toString() + "/test-repo";
+        ZipUtils.unzipToDir(tmpRepo, "/repo.zip");
```

Comment:
Looks good but I'm not a big fan for unzipping it here... Seems to be a bit fragile...

Would it be possible to create a directory in `src/test/resources` with almost empty `.git` and simulate that everything is there?